src/main/java/uk/gov/justice/hmpps/prison/repository/BookingRepository.java

```diff
@@ -515,25 +515,6 @@ private static class OffenderBookingSeq {
         private Integer bookingSeq;
     }
 
-    public Page<ScheduledEvent> getBookingAppointments(final Long bookingId, final LocalDate fromDate, final LocalDate toDate, final long offset, final long limit, final String orderByFields, final Order order) {
```

Comment:
so where has this code gone?

Reply:
It's was only called from endpoints that are not used.  It was called from my last PR (no longer used) but I forgot to remove it